src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -7684,57 +7684,69 @@ void Test16(int val)
+                // (89,18): error CS0150: A constant value is expected
+                //             case y10:
+                Diagnostic(ErrorCode.ERR_ConstantExpected, "y10").WithLocation(89, 18),
+                // (98,17): error CS0103: The name 'y11' does not exist in the current context
+                //         switch (y11 + 11 is var x11 ? x11 : 0)
+                Diagnostic(ErrorCode.ERR_NameNotInContext, "y11").WithArguments("y11").WithLocation(98, 17),
+                // (100,25): error CS0841: Cannot use local variable 'y11' before it is declared
+                //             case 0 when y11 > 0:
+                Diagnostic(ErrorCode.ERR_VariableUsedBeforeDeclaration, "y11").WithArguments("y11").WithLocation(100, 25),
+                // (102,18): error CS0841: Cannot use local variable 'y11' before it is declared
+                //             case y11:
+                Diagnostic(ErrorCode.ERR_VariableUsedBeforeDeclaration, "y11").WithArguments("y11").WithLocation(102, 18),
+                // (102,18): error CS0150: A constant value is expected
+                //             case y11:
+                Diagnostic(ErrorCode.ERR_ConstantExpected, "y11").WithLocation(102, 18),
+                // (112,28): error CS0128: A local variable named 'x14' is already defined in this scope
+                //                   2 is var x14, 
+                Diagnostic(ErrorCode.ERR_LocalDuplicate, "x14").WithArguments("x14").WithLocation(112, 28),
+                // (125,25): error CS0841: Cannot use local variable 'y15' before it is declared
+                //             case 0 when y15 > 0:
+                Diagnostic(ErrorCode.ERR_VariableUsedBeforeDeclaration, "y15").WithArguments("y15").WithLocation(125, 25),
+                // (127,18): error CS0841: Cannot use local variable 'y15' before it is declared
+                //             case y15: 
+                Diagnostic(ErrorCode.ERR_VariableUsedBeforeDeclaration, "y15").WithArguments("y15").WithLocation(127, 18),
+                // (127,18): error CS0150: A constant value is expected
+                //             case y15: 
+                Diagnostic(ErrorCode.ERR_ConstantExpected, "y15").WithLocation(127, 18),
+                // (138,25): error CS0841: Cannot use local variable 'y16' before it is declared
+                //             case 0 when y16 > 0:
+                Diagnostic(ErrorCode.ERR_VariableUsedBeforeDeclaration, "y16").WithArguments("y16").WithLocation(138, 25),
+                // (140,18): error CS0841: Cannot use local variable 'y16' before it is declared
+                //             case y16: 
+                Diagnostic(ErrorCode.ERR_VariableUsedBeforeDeclaration, "y16").WithArguments("y16").WithLocation(140, 18),
+                // (140,18): error CS0150: A constant value is expected
+                //             case y16: 
+                Diagnostic(ErrorCode.ERR_ConstantExpected, "y16").WithLocation(140, 18)
```

Comment:
The addition is all of the "constant value expected" lines. I'd rather keep the alignment that is both consistent with the code and the alignment produced by running the test when it fails.